### ISSUE TITLE
MVS: WP registration pages takes users to Firends and Family name field before completing registration

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#361]
When registering for a WP (any type) after you click the Register Button, focus shifts to the First Name field of the Add Another Person section.  Some times this happens quickly (in a flash) other times the screen with focus on this field will pop up the tip to add the additional persons name.  Then after a short (1 sec or so) period of time the registration completes as normal (no action taken by the user).

During this interlood there is time for the user to click the Register Button again where they will get the message the the person is already registered for the WP.
![vmswpregistrationaddanotherpersonbug](https://cloud.githubusercontent.com/assets/8312206/10765776/a459808c-7c91-11e5-92b3-c8585f80840a.JPG)

**Comment by cewing** [comment]
@WaBirder57, I am unable to reproduce this behavior at all, locally or at staging.wta.org:6543.  What device and browser are you using when you see it?

#### Here's what i've tried:

* registering without entering data in any field (just leaving the pre-filled data from my user profile)
* adding some data in the carpool field so that it was focused when I clicked on 'register'
* clicking on the waiver checkbox last so that was the focused field when clicking the register button.
* attempting to click on the register button a second time while the form is submitting.

#### Here's what I've observed:

In all of these scenarios using Chrome on OS X I can see no change in focus of the form fields and nothing that looks like a tool tip from any of the fields in the "bringing someone with you" section of the form.  

The form does sit for a second or so when I click "register", but even when I try to directly click on the 'register' button a second time during that space, it does not have any effect.  I am unable to get the form to re-submit.  

Any further instructions on how best to reproduce this error would be helpful.

**Comment by cewing** [comment]
In debugging this, I've found another issue that needs fixing.  I will open a separate ticket for that one, as it is only tangentially related to this issue.

**Comment by cewing** [comment]
@WaBirder57 I'm going to assign this to you for re-testing.  If you can reliably reproduce it and tell me how to do so as well, please assign back to me.

**Comment by WaBirder57** [comment]
@cewing I have been able to reproduce this 2 times in a row for a single work party, 2 different users.  I will play around a little more to see if I can  get a better understand what is going on.

Using Firefox 41.0.2
Work Party: http://staging.wta.org:6543/workparty/a2eK0000000wRbV/
Register as Volunteer (one user was an ACL, other Volunteer)

Here are my steps:
Click to join Work party
Scroll to bottom of registration page
Click to accept the Waiver
Click register

I was able to capture the attached screen shot of what I saw for almost 1 sec before it completed the registration and took me to the confirmation screen.  Note that the Bringing Someone with you fields are outlined in red and the popup for the First Name field is showing.

![vmsregistrationpagedefaultfielderror](https://cloud.githubusercontent.com/assets/8312206/10827969/0b2c6434-7e2f-11e5-942f-48a16db5d2b8.JPG)
